install/mac.rst

```diff
@@ -308,13 +374,54 @@ this:
 
 .. code-block:: console
 
-   Brians-Mac:~ brian$ mpf --version
-   MPF v0.50.82
+   My-Mac:~ $ mpf --version
+   MPF v0.51.3
 
 (Note that the actual version number of your MPF installation will be whatever
 version is the latest.)
 
-6. Download & run the "Demo Man" example game
+5.2 Install MPF & MC (Latest)
+-----------------------------
+
+The stable release of MPF is updated every few months, but if you always want the most up-to-date
+changes, which often fix (but sometimes cause) bugs and introduce new features, you can run MPF
+from the latest development build.
+
+The installation instructions are the same, except for including "--pre" in the install command
+(for "prerelease").
+
+.. code-block:: console
+
+   pip install --pre mpf mpf-mc
+
+The prereleases will have "dev" in their version number to indicate that they are under development.
+
+.. code-block:: console
+
+   My-Mac:~ $ mpf --version
+   MPF v0.52.0.dev3
+
+6. Update Kivy to 1.11
+----------------------
+
+There is a known bug in Kivy 1.10 that causes intermittent freezing in applications, including MC.
```

Comment:
They actually released 1.10.1 which contains the fix. That should be our version.

Reply:
Cool, I'll remove this whole section then. One less step!